### ISSUE TITLE
[Organization Proposal] Change namespaces to match PSR-4 and common style

### ISSUE THREAD

**Issue by prowwid** [yiisoft/di#19]
**In [this comment](https://github.com/yiisoft/di/issues/10#issuecomment-358451907) @samdark said that this container isn't Yii-specific.**

So, as I get it, this container should be framework-independent and aimed to be used by developers from different communities.

**And this leads to a question: why namespace naming is Yii-specific?** 

Other first-class frameworks and libraries use upper camel case notation for namespaces so I suggest using upper camel case notation for namespaces in this repository too. 

**Example:**
`Yii\DI\Conatiner`

**Comment by samdark** [comment]
Namespace naming isn't Yii specific. It complies to PSR-2 and PSR-12.

**Comment by prowwid** [comment]
Hello @samdark 

I would rephrase: only Yii namespaces in lower camel case - everyone else use upper camel case. 
So in a code we may have something like:
```php
use Symfony\Component\Process\Process;
use Laravel\Passport\Token;
use GuzzleHttp\Psr7\Request;
use yii\di\Injector;
```
And Yii namespace naming breaks consistency.

**Comment by prowwid** [comment]
Closing this issue as there is no point in further discussion.
